src/main/java/xbot/common/controls/actuators/XCANMotorController.java

```diff
@@ -126,6 +130,8 @@ protected XCANMotorController(
         police.registerDevice(DevicePolice.DeviceType.CAN, busId, info.deviceId(), info.name());
         this.akitName = info.name()+"CANMotorController";
 
+        this.unhealthyAlert = new Alert("Motor Controller " + info.deviceId() + " on CAN bus " + busId.toString() +  " is unhealthy", Alert.AlertType.kError);
```

Comment:
Also I just noticed Alerts can take a 3rd arg (in first position) for a 'group' that will allow us to group these in the AS so we show some but not others. Maybe these should all go in a `"DeviceHealth"` group?